docs/UserGuide.md

```diff
@@ -27,13 +27,20 @@ For more information or to customise your own report, do read up on the followin
 1. Execute it on the OS terminal.  <br/>
 
 ### Using repo-config.csv file
-Usage: `java -jar RepoSense.jar -config CONFIG_DIRECTORY [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]`
+Usage: `java -jar RepoSense.jar [-config CONFIG_DIRECTORY...] [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]`
 1. The report will be generated in the designated OUTPUT_DIRECTORY, or current working directory otherwise.
 
-Sample usage to generate the report:
+Sample usage to generate the report (with -config):
 ```
 java -jar RepoSense.jar -config ./configs/ -output output_path/ -since 01/10/2017 -until 01/11/2017 -formats java adoc js
 ```
+
+Sample usage to generate the report (without -config):
```

Comment:
this section is not necessary. If we do such examples, we would have to do for all the parameters - which is redundant.